### DISCUSSION TITLE
Прошу подсказать логику пересчёта координат в функциях y2lat и x2lon

### DISCUSSION THREAD

**Discussion by Chalokian** [rendrom/rosreestr2coord#80] [General]
Привет. Поясните, плиз, функции из сабжа - это пересчёт из какой системы координат в какую?
Как вообще эта магия работает?

**Comment by rendrom** [comment]
Данные функции выполняют преобразование из меркаторовской проекции в географические координаты (широта и долгота).

y2lat(y): преобразует координату Y из меркаторовской проекции в географическую широту
x2lon(x)  координату X в географическую долготу

Эти преобразования основаны на математических формулах меркаторовской проекции и их обратных вычислениях.
